pyproject.toml

```diff
@@ -172,13 +173,11 @@ run-cov = "echo Code coverage not implemented && exit 1"
 
 [[tool.hatch.envs.hatch-test.matrix]]
 python = [
-    "3.8",
     "3.9",
     "3.10",
     "3.11",
     "3.12",
-    "pypy3.8",
-    "pypy3.9",
```

Comment:
Why did you remove Pypy3.9?

Reply:
It's dead. PyPy now only supports 3.10 (and 2.7)